action.yml

```diff
@@ -43,12 +30,36 @@ inputs:
       HINT: use $ {{ secrets.DOCKERHUB_TOKEN }} (no space after $)
     default: ""
     required: false
+  # OPTIONAL: ghcr.io
+  ghcr_token:
+    description: |
+      IF USING ghcr.io (else, var is ignored):
+        a GitHub token for authenticating with ghcr.io when pushing Docker images
+      HINT: use $ {{ secrets.GITHUB_TOKEN }} (no space after $)
+    default: ""
+    required: true
+  # OPTIONAL: CVMFS
+  gh_cvmfs_token:
```

Comment:
Maybe `cvmfs_gh_token` to make it consistent with other cvmfs options, and clear that this only applies to cvmfs publishing?